packages/react-native-reanimated/src/specs/index.ts

```diff
@@ -0,0 +1,4 @@
+'use strict';
+import ReanimatedTurboModule from './NativeReanimatedModule';
+
+export { ReanimatedTurboModule };
```

Comment:
Will `export { ReanimatedTurboModule } from './NativeReanimatedModule';` also work here?

Reply:
No, `ReanimatedTurboModule` has to be a default export by the rules of the Codegen. This bit of code turns it into a named export.